samples/powerpoint/custom-functions/search-wikipedia-api.yaml

```diff
@@ -0,0 +1,221 @@
+order: 1
+id: powerpoint-custom-functions-search-wikipedia-api
```

Comment:
Custom functions has a very specific meaning in Excel, so we should avoid using that term in this sample (lest people think PowerPoint supports [custom functions](https://docs.microsoft.com//office/dev/add-ins/excel/custom-functions-overview)).

I'd recommend changing the category and folder to "Scenarios". This sample shows the common scenario of calling an external web API for data.

Reply:
Move and update to scenarios folder